src/exposition/http.rs

```diff
@@ -248,6 +270,75 @@ mod handlers {
         Ok(content)
     }
 
+    pub async fn binary_metadata() -> Result<impl warp::Reply, Infallible> {
+        let mut metadata = Metadata::new();
+
+        for metric in &metriken::metrics() {
```

Comment:
This should be true but just verifying: does iteration order remains consistent across lifetime of the application?

Reply:
Iteration order is now consistent since we guarantee we are only exposing static metrics, the ordering in that slice is determined at link time. Comments added.